extensions/native/tests/programs/Cargo.toml

```diff
@@ -18,3 +19,7 @@ openvm-native-transpiler = { path = "../../transpiler", default-features = false
 openvm-instructions = { path = "../../../../crates/toolchain/instructions", default-features = false }
 openvm-stark-sdk = { git = "https://github.com/openvm-org/stark-backend.git", rev = "8c9f94", default-features = false }
 openvm-stark-backend = { git = "https://github.com/openvm-org/stark-backend.git", rev = "8c9f94", default-features = false }
+
+[[example]]
```

Comment:
test should not be in `extensions/native`. it should be in `extensions/rv32im` or `crate/toolchain/tests`